master/buildbot/changes/gitpoller.py

```diff
@@ -340,6 +349,7 @@ def _process_changes(self, newRev, branch):
             dl = defer.DeferredList([
                 self._get_commit_timestamp(rev),
                 self._get_commit_author(rev),
+                self._get_commit_committer(rev),
                 self._get_commit_files(rev),
                 self._get_commit_comments(rev),
```

Comment:
Probably we could get timestamp, author and commit in only one command but this is not a big deal

Reply:
Do you want me to combine them?

Reply:
I think it's worth doing that as a separate PR. The less things in a single PR the less chance that it'll break.